.git-blame-ignore-revs

```diff
@@ -4,3 +4,6 @@
 
 # Format all files with Prettier
 2f3b359987bebf00d45a8c32d58e3b302d117c4a
+
+# Format Python files with black + isort
+61dea371246bb7e14e42649746187c9e0628ba1b
```

Comment:
@nwalters512 can confirm, this is the latest commit hash for running the formatter:

![image](https://user-images.githubusercontent.com/1345068/211723954-c958fbcb-1961-47ef-be4e-65e8d37c3fc8.png)